packages/server/db/knex.ts

```diff
@@ -18,12 +18,7 @@ config.log = {
 
 dbStartupLogger.info(`Loaded knex conf for ${env}`)
 
-/**
- * Need to override type because type def file incorrectly uses ES6
- * @type {import('knex').default}
- */
-const knex = require('knex')
 const knexInstance = knex(config)
```

Comment:
certainly doable, but not a trivial change so should be handled separately. it would involve a `getKnex()` fn that creates it and stores it in module scope on initial invocation, which means that a ton of files would require changes (replace `knex` w/ `getKnex()`)

Reply:
also - many of the factories we currently build are built as a sideeffect in the module scope so those would have to be changed too